forem/forem
# Update data exporter to handle admin send

## What type of PR is this? (check all applicable)

## Description
This updates the data exporter functionality to handle being able to send to admins. We want to support this use-case in the event that someone needs to be deleted from the admin's end and we want to send them their data before removing them.

Other technical changes include:
- updating the code so that you _always_ send an email -- previously the code had the option to not send an email, but I don't think that was ever used
- adding a new test for sending to the admin

## QA Instructions, Screenshots, Recordings
![Screen Shot 2020-09-10 at 12 39 48 PM](https://user-images.githubusercontent.com/17884966/92764781-be1ae680-f362-11ea-85b8-99480ba7209f.png)

QA Steps:
1. Go to any user `/admin/users/1/edit`
2. Hit send to admin or user
3. Go to `localhost:3000/sidekiq` to see if a job was queued

## Added tests?

## Added to documentation?

## [optional] Are there any post deployment tasks we need to perform?
Nope, just need to make sure it works in production.
